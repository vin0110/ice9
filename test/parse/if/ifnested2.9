if a -> if b -> c; [] else -> d; fi fi
