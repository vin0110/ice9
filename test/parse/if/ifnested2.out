IF:
  ID:'a'
(then)
  IF:
    ID:'b'
  (then)
    EXP:
      ID:'c'
  (else)
    EXP:
      ID:'d'
(else)
  nil
